servers/src/test/java/tachyon/worker/block/TieredBlockStoreTestUtils.java

```diff
@@ -32,37 +38,44 @@
 import tachyon.worker.block.meta.TempBlockMeta;
 
 /**
- * This class provides utility methods for testing tiered block store.
+ * This class provides utility methods for setting and testing tiered block store.
  */
 public class TieredBlockStoreTestUtils {
   /**
    * Default configurations of a TieredBlockStore for use in {@link #defaultMetadataManager}. They
    * represent a block store with a MEM tier and a SSD tier, there are two directories with capacity
-   * 100 bytes and 200 bytes separately in the MEM tier and three directories with capacity 1000,
-   * 2000, 3000 bytes separately in the SSD tier.
+   * 2000 bytes and 3000 bytes separately in the MEM tier and three directories with capacity 10000,
+   * 20000, 30000 bytes separately in the SSD tier.
    */
   public static final int[] TIER_LEVEL = {0, 1};
   public static final StorageLevelAlias[] TIER_ALIAS = {StorageLevelAlias.MEM,
       StorageLevelAlias.SSD};
   public static final String[][] TIER_PATH =
       { {"/mem/0", "/mem/1"}, {"/ssd/0", "/ssd/1", "/ssd/2"}};
-  public static final long[][] TIER_CAPACITY = { {2000, 3000}, {10000, 20000, 30000}};
+  public static final long[][] TIER_CAPACITY_BYTES = {{2000, 3000}, {10000, 20000, 30000}};
+  public static final String WORKER_DATA_FOLDER = "/tachyonworker/";
+
+  public static TachyonConf sTachyonConf = WorkerContext.getConf();
 
   /**
-   * Create a {@link TachyonConf} for a {@link TieredBlockStore} configured by the parameters. For
-   * simplicity, you can use {@link #defaultTachyonConf(String)} which calls this method with
-   * default values.
+   * Setup a {@link TachyonConf} for a {@link TieredBlockStore} with several tiers configured
```

Comment:
style: As per http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide, use 3rd person, not 2nd person. So ``Sets up`` instead of ``Setup``.